public/content/contributing/design/index.md

```diff
@@ -30,7 +30,7 @@ Provide feedback on our website by:
 
 ### <Emoji text=":three:" size={1} /> &nbsp;Find design related issues on the website and report them {#report-design-issues}
 
-Ethereum.org is a fast growing website with many features and content. Some of the UI can easily become obsolete or could be improved. If you encounter any such instance, please report it so that it gets our attention.
+ethereum.org is a fast growing website with many features and content. Some of the UI can easily become obsolete or could be improved. If you encounter any such instance, please report it so that it gets our attention.
```

Comment:
Hm... Looking at this, we use a split throughout the repo when we start a sentence with the domain/name... about 47 use lowercase, and about 99 capitalize.

Personally think starting a sentence with a lowercase letter always looks awkward and would advocate for using "Etheruem.org" in these cases.

@minimalsm Thoughts here?

```suggestion
Ethereum.org is a fast growing website with many features and content. Some of the UI can easily become obsolete or could be improved. If you encounter any such instance, please report it so that it gets our attention.
```

Reply:
I thought `ethererm.org` is started with always lowercase.
Look like this page https://ethereum.org/en/about/.
Similar to iPad and iPhone. 
If we use `IPhone is very cool` looks awkward.

Reply:
Or maybe it needs to be aligned with Ethereum.org as a proper noun for preventing this situation.

Reply:
I opened the discussion about this.

https://github.com/ethereum/ethereum-org-website/discussions/12669

Reply:
Great thanks! We don't usually use GitHub "Discussions" so I'm honestly not sure how much traction it'll get over there.

This isn't something I personally have a strongly held opinion on, was just noting above that we more commonly use "Ethereum.org" over "ethereum.org" when starting a sentence. I tend to prefer consistency above all.

I agree with your point about how "iPhone" is used, not "IPhone", but still feels a little strange I guess, but not a big deal.

cc: @minimalsm Did you have any thoughts on this? We have always tended to use the lowercase domain as the official name, so I wouldn't be against just formalizing that and keeping that pattern even when starting a sentence.